tasks/size_contracts.js

```diff
@@ -13,6 +13,23 @@ const formatSize = function (size) {
   return (size / 1024).toFixed(3);
 };
 
+const ansiRegex = function ({ onlyFirst = false } = {}) {
```

Comment:
What is `onlyFirst` used for?

Reply:
Only the first occurrence of ANSI will be identified not global
Since the code was sourced from a util I left it in, not required here
Shall I to remove it?

Reply:
Well the idea was to simply add `strip-ansi` as a dependency but both of these are ES modules, would it be better to set up babel and proceed with `import`?

Reply:
**Update:** Found an older CommonJS compatible version, can be kept for now until you decide to upgrade